lockbox-ios/Common/Resources/Constants.swift

```diff
@@ -8,10 +8,11 @@ import UIKit
 
 struct Constant {
     struct app {
+        static let appVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String
         static let redirectURI = "https://mozilla-lockbox.github.io/fxa/ios-redirect.html"
         static let faqURL = "https://lockbox.firefox.com/faq.html"
         static let privacyURL = "https://lockbox.firefox.com/privacy.html"
-        static let provideFeedbackURL = "https://qsurvey.mozilla.com/s3/Lockbox-Input"
+        static let provideFeedbackURL = "https://qsurvey.mozilla.com/s3/Lockbox-Input?ver=" + appVersion!
```

Comment:
nit: I think the swift-y way is to do something like: `"https://qsurvey.mozilla.com/s3/Lockbox-Input?ver=\(appVersion ?? "1.0")"`

Reply:
Ah ok, and a nice fallback. Thanks 👍